parts/linux/cloud-init/artifacts/cloud-init-complete.target

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Custom Target for Cloud Init Completion
```

Comment:
we dont really need this, you can do a `cloud-init status --wait` to ensure that cloud-init has finished before running anything else

Reply:
was able to get it working without this target by using systemd dependencies with cloud-init.target and cloud-final.service :)